src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -153,6 +150,51 @@ public void getTagsToExcludeFromChart_ReturnsEmptyList() {
         assertThat(patterns).isEmpty();
     }
 
+    @Test
+    public void getDirectorySuffix_ReturnsDirectorySuffix() {
+
+        // given
+        String directorySuffix = "test";
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        configuration.setDirectorySuffix(directorySuffix);
+
+        // then
+        assertThat(configuration.getDirectorySuffix()).isEqualTo(directorySuffix);
+    }
+
+    @Test
+    public void getDirectorySuffixWithSeparator_ReturnsDirectorySuffixWithSeparator() {
+
+        // given
+        String directorySuffix = "test";
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        configuration.setDirectorySuffix(directorySuffix);
+
+        // then
+        Class<Configuration> configClass = Configuration.class;
+        try {
+            Field suffixSeparator = configClass.getDeclaredField("SUFFIX_SEPARATOR");
```

Comment:
I don't get it why this must be so complicated :)

Reply:
I didn't want to hardcode the separator - so had to get the value of a private variable.  Anyway, I changed it now to have a hardcoded '_' separator in the test.